src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -322,11 +322,6 @@ void UpdateLeftToolbarItems()
 				image = _context.Shell.FlyoutIcon;
 			}
 
-			if (!IsRootPage)
-			{
-				NavigationItem.HidesBackButton = !backButtonVisible;
```

Comment:
What if all we did was set the image to null here if backButtonVisible is false?

```C#
if (!IsRootPage)
			{
				NavigationItem.HidesBackButton = !backButtonVisible;
				image = (backButtonVisible) ? image : null;
			}
```